arangod/RestHandler/RestAdminLogHandler.cpp

```diff
@@ -472,3 +478,63 @@ void RestAdminLogHandler::handleLogLevel() {
                   TRI_ERROR_HTTP_METHOD_NOT_ALLOWED);
   }
 }
+
+void RestAdminLogHandler::handleLogStructuredParams() {
+  std::vector<std::string> const& suffixes = _request->suffixes();
+
+  // was validated earlier
+  TRI_ASSERT(!suffixes.empty());
+
+  if (suffixes[0] != "structured") {
+    generateError(rest::ResponseCode::BAD, TRI_ERROR_HTTP_SUPERFLUOUS_SUFFICES,
+                  "superfluous suffix, expecting /_admin/log/structured");
+    return;
+  }
+
```

Comment:
This shouldn't be done here, but in the caller.

Reply:
Addressed